### DISCUSSION TITLE
BIC branch code 619 for 8 character length codes.

### DISCUSSION THREAD

**Discussion by krzysztofmeler** [Simplify/ibantools#131] [Q&A] [answered]
Hi, in commit 73d38ff42e0e8023578ea6f95f891fff8ce48a12 there is a change in extraction of BIC. Instead of setting branch code to null, it is set to '619' for whatever reason:
https://github.com/Simplify/ibantools/commit/73d38ff42e0e8023578ea6f95f891fff8ce48a12#diff-a7ba2852158b3ce7f694a64076a658eb3e223f80a71783d4e4ce10227f057361R258
What is the reason for this change, is this 619 code part of the BIC spec? I thought that XXX is used sometimes for BICs without branch code, but I can't find information about 619.

**Comment by Simplify** [comment, marked as answer]
Can't remember what was my thinking then. It's definitely a bug.

**Comment by Simplify** [comment]
Fixed in 4.1.6
